### DISCUSSION TITLE
Installing Python packages

### DISCUSSION THREAD

**Discussion by seraph776** [codewars/codewars.com#2839] [Q&A] [answered]
I was trying to create a `Kumite` using `requests-html` module.  The code works on my machine, however, I received a `ModuleNotFoundError: No module named 'requests_html'` (screenshot below).

![image](https://user-images.githubusercontent.com/72005563/196840949-4b13503f-e04d-43dd-8211-3cdb782c9ef7.png)

Is there a way for the user to install python packages that are required to run scripts? Is there something I am doing wrong? 

Thanks ,

**Comment by Blind4Basics** [comment, marked as answer]
This module effectively isn't installed. There is no simple (valid) way to install additional packages at runtime and most likely, that would consume most of the allowed computation time anyway. You're not doing anything wrong, but you just cannot use that.

Cheers
